tiledb/sm/array_schema/enumeration.cc

```diff
@@ -450,3 +444,17 @@ void Enumeration::add_value_to_map(std::string_view& sv, uint64_t index) {
 }
 
 }  // namespace tiledb::sm
+
+std::ostream& operator<<(
+    std::ostream& os, const tiledb::sm::Enumeration& enumeration) {
+  os << "### Enumeration ###" << std::endl;
+  os << "- Name: " << enumeration.name() << std::endl;
+  os << "- Loaded: true" << std::endl;
```

Comment:
"Loaded" isn't part of the enumeration definition, but a hack for deferred loading of the enumeration data.